server/app.ts

```diff
@@ -529,7 +529,10 @@ export class Application implements ServerApplication {
 
   /** returns the route module by given module. */
   private getRouteModule({ url, hash }: Pick<Module, 'url' | 'hash'>): RouteModule {
-    const useDeno = this.lookupDeps(url).filter((({ url }) => url.startsWith('#useDeno-'))).length > 0 || undefined
+    let useDeno = this.lookupDeps(url).filter((({ url }) => url.startsWith('#useDeno-'))).length > 0 || undefined
+    if (this.config.ssr === false) {
+      useDeno = undefined
```

Comment:
Small refactor - you can use `const` with the ternary operator, its a little cleaner.